client/src/App.js

```diff
@@ -1,26 +1,32 @@
-import { useState } from 'react';
-import './App.css';
+import TasteCard from './components/TasteCard';
+import { tastes } from './database';
+import styled from 'styled-components';
 
 function App() {
-  const [users, setUsers] = useState([]);
-
-  function fetchSomething() {
-    fetch('/api')
-      .then(res => res.json())
-      .then(data => setUsers(data));
-  }
-
   return (
-    <div className="App">
-      <h1>Users</h1>
-      <button onClick={fetchSomething}>Fetch users</button>
-      <ul>
-        {users.map(user => (
-          <li key={user.id}>{user.name}</li>
+    <AppContainer>
+      <h1>Coffee Roast Levels</h1>
+      <CardWrapper>
+        {tastes.map((taste, id) => (
+          <TasteCard key={id} taste={taste} />
         ))}
-      </ul>
-    </div>
+      </CardWrapper>
+    </AppContainer>
   );
 }
 
 export default App;
+
+const AppContainer = styled.main`
+  background-color: var(--color-light-gray);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const CardWrapper = styled.ul`
```

Comment:
When you're using a `ul` here the `CardWrapper` should be a `li` element and not a `article`